src/main/resources/db/migration/V1_0__CreateTables.sql

```diff
@@ -0,0 +1,36 @@
+create table AUTHORENTITY
```

Comment:
@venividiviciregina Please merge master and rework SQL code according to changes in master